.husky/pre-push

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+yarn lint-warn
```

Comment:
Note that while the pre-commit hook will run `lint-staged` and automatically include the fixes from eslint and prettier in the commit, this will only warn. I can see an issue here if one only wants to push a subset of changes and there are issues in the changes not being pushed. Any ideas?

Reply:
In that scenarios, I would either use `--no-verify` or temporarily `git stash` the changes that are not being pushed.